kolibri/plugins/learn/assets/src/views/LibraryPage/index.vue

```diff
@@ -190,14 +207,15 @@
 
       const { currentCardViewStyle } = useCardViewStyle();
 
+      const { baseurl, fetchDevices } = useDevices();
```

Comment:
It would be good to keep a consistent camelCase for the variable name

Reply:
My only reason for using `baseurl` like this is that it matches the get parameter used by the API - which means when we pass the value to an API call we can just do `{ baseurl }` rather than remapping.